torch/onnx/utils.py

```diff
@@ -1660,6 +1661,50 @@ def _export(
     return torch_out
 
 
+@_beartype.beartype
+def _add_onnxscript_fn(
+    model_bytes: bytes, graph: _C.Graph, custom_opsets: Mapping[str, int]
+) -> bytes:
+    """Insert model-included custom onnx-script function into ModelProto"""
+
+    # TODO(titaiwang): remove this when onnx becomes dependency
+    try:
+        import onnx
+    except Exception:
+        raise errors.OnnxExporterError("Module onnx is not installed!")
+
+    onnx_function_list = list()
+    # Iterate graph nodes to insert only the included custom
+    # function_proto into model_proto
+    for node in graph.nodes():
+        if symbolic_helper._is_custom_domain(node):
+            domain_op = node.kind()
+            domain, opname = domain_op.split("::")
```

Comment:
I am seeing more places where we need to split the string. I would consider creating a util somewhere (symbolic_helpers?)

If we decided symbolic_helpers is the place, it needs to be public as symbolic_helpers itself is public and we don't want to use private functions in a module where it is not defined.

Reply:
#87638 
Personally I feel like this function is too easy that no external users would care. I suugest if there is function that we are not sure about going public, we just make it internal first. What do you think?

Reply:
SG. Commented in the PR!